virt-density.go

```diff
@@ -54,6 +57,7 @@ func NewVirtDensity(wh *workloads.WorkloadHelper) *cobra.Command {
 	}
 	cmd.Flags().IntVar(&vmsPerNode, "vms-per-node", 245, "VMs per node")
 	cmd.Flags().DurationVar(&vmiRunningThreshold, "vmi-ready-threshold", 25*time.Second, "VMI ready timeout threshold")
+	cmd.Flags().StringVar(&vmImage, "vm-image", "quay.io/rsevilla/cirros:0.6.3", "Vm Image to be deployed")
```

Comment:
@jeniferh should we update this to the fc40 image?

Reply:
oh yeah, should match the other change here: https://github.com/kube-burner/kube-burner-ocp/pull/186/files#diff-e2c84da9911338751c80993fd5ae1735104d5880c071370f8441b12cff5b8960R79

Reply:
I am okay with that. I did want to change it  to not disturb the results of the CI by changing the default image that is probably used elsewhere.